redbeams/src/main/java/com/sequenceiq/redbeams/domain/DatabaseServerConfig.java

```diff
@@ -256,14 +261,19 @@ public void setEnvironmentId(String environmentId) {
         this.environmentId = environmentId;
     }
 
+    public Set<DatabaseConfig> getDatabases() {
+        return databases;
+    }
+
     /**
-     * Creates a database with a database name and type from the database server config.
+     * Creates a database based on this database server config.
      *
      * @param databaseName the name of the database
-     * @param type         the type of database
+     * @param type         the type of the database
+     * @param status       the resource status of the database
      * @return a DatabaseConfig
      */
-    public DatabaseConfig createDatabaseConfig(String databaseName, String type) {
+    public DatabaseConfig createDatabaseConfig(String databaseName, String type, ResourceStatus status) {
```

Comment:
This is not a bad place for this method but I would move it into DatabaseServerConfigService.

Reply:
Fair enough. This is where @meowfaceman put it and moving it would be out of scope here.